examples/README.md

```diff
@@ -0,0 +1 @@
+Note: When using VSCode, if you would like to use rust-analyzer for the examples, you need to set `rust-analyzer.cargo.features=true` for `examples`.
```

Comment:
please remove this -- let me try to fix this at another level by adding arkworks to the dev-dependencies for now.